src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/user-groups-list.js

```diff
@@ -0,0 +1,99 @@
+angular.module('virtoCommerce.DemoSolutionFeaturesModule')
+    .controller('virtoCommerce.DemoSolutionFeaturesModule.userGroupsListController', ['$scope', 'platformWebApp.bladeNavigationService', 'virtoCommerce.DemoSolutionFeaturesModule.userGroupsApi',
+        function ($scope, bladeNavigationService, userGroupsApi) {
+            var blade = $scope.blade;
+            blade.currentEntity = undefined;
+
+            userGroupsApi.taggedMember({ id: blade.memberId },
+                function(result) {
+                    blade.currentEntity = result || {};
+                    blade.currentEntity.tags = blade.currentEntity.tags || [];
+                    blade.isLoading = false;
+            });
+
+            blade.toolbarCommands = [
+                {
+                    name: "platform.commands.add",
+                    icon: 'fa fa-plus',
+                    executeMethod: () => {
+                        bladeNavigationService.closeChildrenBlades(blade, () => {
+                            var newBlade = {
+                                id: 'assignUserGroup',
+                                title: 'demoSolutionFeaturesModule.blades.assign-user-group.title',
+                                subtitle: 'demoSolutionFeaturesModule.blades.assign-user-group.subtitle',
+                                controller: 'virtoCommerce.DemoSolutionFeaturesModule.assignUserGroupController',
+                                template: 'Modules/$(VirtoCommerce.DemoSolutionFeaturesModule)/Scripts/blades/assign-user-group.tpl.html',
+                                onGroupsAdded: (entity) => {
+                                    const newTags = _.difference(entity.tags, blade.currentEntity.tags);
+                                    blade.currentEntity.tags = [...blade.currentEntity.tags, ...newTags];
+                                    bladeNavigationService.closeChildrenBlades(blade);
+                                    saveChanges();
+                                }
+                            };
+                            bladeNavigationService.showBlade(newBlade, blade);
+                        });
+                    },
+                    canExecuteMethod: () => true
```

Comment:
Maybe we should use customer permission to manipulate member tags?

Reply:
done